canonical-websites/insights.ubuntu.com
# Hardcode routes, remove local data

**Based on #205 - review that first**

Replace:
- "/topics/[slug]" with explicit topic route ("/topics/maas" etc.)
- "/[group_slug]" with explicit group pages ("/cloud-and-server" etc.)

This is because each of these pages has hard-coded data. And since they
appear explicitly in the navigation, they are clearly intended to be
dedicated pages, rather than dynamic pages.

This structure is more declarative and much clearer to work with.

And it means we can remove all the local data.

QA
--

`./run`

Go to:

- http://0.0.0.0:8023/press-centre
- http://0.0.0.0:8023/cloud-and-server
- http://0.0.0.0:8023/desktop
- http://0.0.0.0:8023/internet-of-things
- http://0.0.0.0:8023/topics/maas
- http://0.0.0.0:8023/topics/design
- http://0.0.0.0:8023/topics/juju
- http://0.0.0.0:8023/topics/snappy

Each should look exactly as it used to (see https://insights.staging.ubuntu.com for reference)
